packages/test/test-end-to-end-tests/src/test/gc/gcSweepDataStores.spec.ts

```diff
@@ -318,6 +345,10 @@ describeNoCompat("GC data store sweep tests", (getTestObjectProvider) => {
 					eventName: "fluid:telemetry:ContainerRuntime:GC_Deleted_DataStore_Requested",
 					clientType: "interactive",
 				},
+				{
+					eventName: "fluid:telemetry:ContainerRuntime:GC_Deleted_DataStore_Requested",
```

Comment:
These events are incorrectly named. They are logged because deleted data stores receives an op (or signal in the next test). This had me perplexed for a while!
I will fix them in a separate PR.

Reply:
AB#6520